.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea/
```

Comment:
Can you remove this change? You should add a [global .gitignore](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/) for this.

Reply:
I removed this change.